recipes/cunit/all/conanfile.py

```diff
@@ -50,10 +50,14 @@ def requirements(self):
         if self.options.with_curses == "ncurses":
             self.requires("ncurses/6.2")
 
+    @property
+    def _settings_build(self):
+        return getattr(self, "settings_build", self.settings)
+
     def build_requirements(self):
-        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/20200517")
         self.build_requires("libtool/2.4.6")
+        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            self.build_requires("msys2/cci.latest")
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
```

Comment:
Github wont let me but you also need to
- remove the next line
- add this https://github.com/conan-io/conan-center-index/blob/cd325616aab3f9373a450f32805fbbcd50f901f3/recipes/pdqsort/all/conanfile.py#L3

```suggestion
        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```